CodingChallenges/CC_90_dithering_p5.js/sketch.js

```diff
@@ -0,0 +1,93 @@
+let kitten;
+
+function preload() {
+  kitten = loadImage("data/kitten.jpg");
+}
+
+function setup() {
+  createCanvas(1024, 512);
+
+  image(kitten, 0, 0);
+  makeDithered(kitten, 1);
+  image(kitten, 512, 0);
+  // Apply gray filter to the whole canvas
+  filter(GRAY);
+}
+
+function imageIndex(img, x, y) {
+  return 4 * (x + y * img.width);
+}
+
+function getColorAtindex(img, x, y) {
+  const idx = imageIndex(img, x, y);
```

Comment:
I tend to not use `const` in my examples to be more beginner friendly, but I don't feel super strongly about this. Does it matter much?

Reply:
I just use const by default, no problem changing them all to let instead.